components/plate_based_widgets/mapeditor/PlateHeatMap.vue

```diff
@@ -11,7 +11,6 @@
         :id="'column_' + column_index"
         @click.exact="on_column_select(column_values[column_index - 1])"
         @click.shift.exact="on_column_ctrl_click_or_shift_click(column_values[column_index - 1])"
-        @click.ctrl.exact="on_column_ctrl_click_or_shift_click(column_values[column_index - 1])"
```

Comment:
Let me know if you want to keep the control-click functionality, otherwise, I've just kept the shift-click

Reply:
It didn't work for me so I say get rid of it